src/icons/tasks/IconTaskGeneral.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it is not purely decorative for accessibility */
+    title?: string,
+    width?: number,
+};
+
+const IconTaskGeneral = ({ className = '', color = '#26C281', height = 30, title, width = 30 }: Props) => (
```

Comment:
It looks like these icons are using different default dimensions and viewbox values than most others. Is that intentional?

Reply:
yes it was intentional.